pycbc/pnutils.py

```diff
@@ -518,6 +518,11 @@ def _get_imr_duration(m1, m2, s1z, s2z, f_low, approximant="SEOBNRv4"):
         # NB for no clear reason this function has f_low as first argument
         time_length = lalsimulation.SimIMRSEOBNRv4ROMTimeOfFrequency(
                            f_low, m1 * lal.MSUN_SI, m2 * lal.MSUN_SI, s1z, s2z)
+    elif approximant == 'SPAtmplt' or approximant == 'TaylorF2':
+        chi = lalsimulation.SimInspiralTaylorF2ReducedSpinComputeChi(
+                           m1,m2,s1z,s2z)
+        time_length = lalsimulation.SimInspiralTaylorF2ReducedSpinChirpTime(
+                           f_low, m1 * lal.MSUN_SI, m2 * lal.MSUN_SI, chi, -1)              
```

Comment:
Hi @AlexisMV, could you do a couple more whitespace fixes to address the codeclimate and codebeat complains:
* `m1,m2,s1z,s2z` -> `m1, m2, s1z, s2z`
* Indent with 8 spaces (two tabs)
* Remove the trailing white space after the function call
Also please rebase on master, I think this is needed after Alex's changes to the test suite.

Reply:
Hi @titodalcanton , thanks a lot, I was not sure what it was asking from me. I tried git rebase master and it says the branch is up to date, not sure if I need to do something else (Sorry, still learning git).